noir-projects/aztec-nr/aztec/src/keys/getters.nr

```diff
@@ -54,11 +90,11 @@ fn get_master_key(
     }
 }
 
-fn fetch_key_from_registry(
+fn get_registry_with_key_index(
```

Comment:
I don't like much that we need to have this getter and then split the flows into `fetch_key_from_registry` and `fetch_key_from_registry_at` but I don't really now how to refactor it so will shut up for now.

But I think a better name would help here because when I saw the function I though it fetches some value from registry. I think something like `instantiate_registry_getter` would be a better name.

Reply:
**Update**: Boss came up with how to refactor it [here](https://github.com/AztecProtocol/aztec-packages/issues/6589).